packages/test/test-service-load/src/utils.ts

```diff
@@ -85,7 +80,7 @@ class FileLogger implements ITelemetryBufferedLogger {
 
 	private constructor(
 		private readonly baseLogger?: ITelemetryBufferedLogger,
-		public readonly minLogLevel?: LogLevel,
+		public readonly minLogLevel: LogLevel = LogLevel.verbose,
```

Comment:
nit: I'd remove this from the constructor and hardcode it to a readonly field.  The constructor is private and you just removed the usages, so just keep it simple.